app/templates/events/view.hbs

```diff
@@ -1,6 +1,38 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid">
+    <div class="row">
+      <div class="eleven wide column">
+        <h2 class="ui header">{{t 'Fossasia Test'}}</h2>
+      </div>
+      <div class="five wide column">
+        <button class="ui button small">{{t 'Preview'}}</button>
```

Comment:
<img width="1280" alt="screen shot 2017-05-26 at 11 32 17 am" src="https://cloud.githubusercontent.com/assets/12716067/26482810/108cb92c-4207-11e7-9e7e-41c8ad7d2388.png">

I am not getting any extra space